bindings/rust/extended/s2n-tls-sys/build.rs

```diff
@@ -257,6 +257,9 @@ impl External {
     fn link(&self) {
         println!("cargo:rustc-cfg={EXTERNAL_BUILD_CFG_NAME}");
 
+        // Propagate an external build flag to dependents.
+        println!("cargo:external_build=true");
```

Comment:
I think I'm missing something-- where is this used?

Reply:
This is what sets the environment variable that's read by the s2n-tls build.rs:
https://github.com/aws/s2n-tls/blob/ce5d6848b0ce58b0e34f67e51c3216e3f9118e7e/bindings/rust/extended/s2n-tls/build.rs#L4

> https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key
if the package `foo` depends on `bar`, which links `baz`, then if `bar` generates `key=value` as part of its build script metadata, then the build script of `foo` will have the environment variables `DEP_BAZ_KEY=value`